pgo/pgo.py

```diff
@@ -208,7 +208,9 @@ async def run(command: list[str], cpuset: Optional[str] = None, **kwargs) -> Pro
 
     orig_cmd = cmd[:] # Copy the command here for the purpose of logging, before it's uglified up by the wrappers below.
 
-    kwargs.setdefault("process_group", 0)
+    downstream_kwargs = dict()
```

Comment:
I'm confused what problem you're fixing here. Here "kwargs" doesn't have "a default", Python sets is from scratch every time, no?